### PR TITLE
Add new markdown format option

### PR DESCRIPTION
**Pull request by Pustur**
Hey, awesome job on the format option you just released!

I think that markdown is a really nice format to have, so I decided to implement it.

I have published a gist that shows the output here: ~https://gist.github.com/Pustur/9d5c82372b6fe45686f54e2ea84399e8~

**EDIT:** Noticed there's no documentation about this format option yet, working on it now.

**EDIT2:** I added documentation about the new option, but I also included a summary table at the top to quickly recap the parameters. Feel free to ask for changes if you don't like something.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -111,6 +111,8 @@ async function locatePackages(depth) {
 
 function getRenderer(format) {
   switch (format) {
+    case 'md':
+      return require('./render/markdown.js');
     case 'html':
       return require('./render/html.js');
     case 'console':
```
Comment (Pustur) [inline index.js:118]: PS: can I remove this `case: 'console':` and just leave the default one? @mjswensen

Comment (mjswensen) [reply index.js:118]: I like to leave it there as a tiny piece of documentation for future readers that `console` is both an expected value and the default value.

**Review by mjswensen — APPROVED** [review]
This is fantastic, @Pustur ! I was wanting to add a markdown format option _and_ update the documentation on the README, so it's like you read my mind.

**Comment by mjswensen** [comment]
This has been released at `shoulders@2.3.0`.

**Pull request merged**